### DISCUSSION TITLE
How to write good pull request ?

### DISCUSSION THREAD

**Discussion by Chandra-Sekhar-Bala** [oppia/oppia-android#4401] [Q&A (Other)] [answered]
I am new to this repository. Can someone help me write a good pull request? I mean mentioning the issue and all the other parameters I need to be careful !!

**Comment by BenHenning** [comment, marked as answer]
Hi @Chandra-Sekhar-Bala. Have you already taken a look at https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#step-2-create-a-pr-on-github? If so, what additional parts of the PR template or PR creation process would you like feedback on?

**Reply by Chandra-Sekhar-Bala** [reply]
Sorry my bad, I should've taken a good look on the contributors page. As it's already available I'm really sorry to create a unnecessary discussion here. I'll not repeat that again.
P.S: I'll try to find my answer in the repo, if not found only then move toward mentor or discussion.

**Reply by BenHenning** [reply]
No problem @Chandra-Sekhar-Bala. Asking questions is what this discussion board is for. :) Searching for a solution is always a good first start, though, as it's possible your question has already been asked or there is documentation that could answer it. Feel free to ask more questions as discussions if you need to, though.

**Reply by Chandra-Sekhar-Bala** [reply]
Thanks for the guide @BenHenning, will definitely follow this  :)
